src/lib/openapi/validate.ts

```diff
@@ -12,13 +11,15 @@ const ajv = new Ajv({
     schemas: Object.values(schemas).map((schema) =>
         omitKeys(schema, 'components'),
     ),
-});
-
-addFormats(ajv, ['date-time']);
 
-// example was superseded by examples in openapi 3.1, but we're still on 3.0, so
-// let's add it back in!
-ajv.addKeyword('example');
+    // example was superseded by examples in openapi 3.1, but we're still on 3.0, so
+    // let's add it back in!
+    keywords: ['example'],
+    formats: {
+        'date-time': true,
+        uri: true,
+    },
+});
```

Comment:
This is just a simplification of how we write it. Instead of using the `add*` methods, we can specify things in the options.